app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -732,7 +732,7 @@ class MainActivity :
                 if (viewModel.waitingForSignInToShowStories) {
                     showStories(StoriesSource.USER_LOGIN)
                     viewModel.waitingForSignInToShowStories = false
-                } else if (!settings.getEndOfYearShowModal()) {
+                } else if (settings.getEndOfYearShowModal()) {
```

Comment:
Noticed I didn't remove the negation here when refactoring the method to show EoY modal:
https://github.com/Automattic/pocket-casts-android/pull/1503/commits/ec7708f7c4d418575740bf90ad5fdbaaae0b37ab#diff-fb37a631ad6fd454ee69493af94665cb42e3c57505418ab562222640ff9d843cR728